include/config/battle.h

```diff
@@ -152,7 +152,7 @@
 #define B_X_ITEMS_BUFF              GEN_LATEST // In Gen7+, the X Items raise a stat by 2 stages instead of 1.
 #define B_MENTAL_HERB               GEN_LATEST // In Gen5+, the Mental Herb cures Taunt, Encore, Torment, Heal Block, and Disable in addition to Infatuation from before.
 #define B_TRAINERS_KNOCK_OFF_ITEMS  TRUE       // If TRUE, trainers can steal/swap your items (non-berries are restored after battle). In vanilla games trainers cannot steal items.
-#define B_RESTORE_HELD_BATTLE_ITEMS TRUE       // In Gen9 all non berry items are restored after battle.
+#define B_RESTORE_HELD_BATTLE_ITEMS TRUE       // In Gen9, all non-berry items are restored after battle.
```

Comment:
It's used on a boolean `TRUE`/`FALSE` basis in the code though, which makes sense for a On/Off toggle like this imo.

Reply:
It does mention Gen9 directly though, it should be "If TRUE, all non-berry items are restored after battle." if you want to keep it a bool

Reply:
> It does mention Gen9 directly though, it should be "If TRUE, all non-berry items are restored after battle." if you want to keep it a bool

That's fair. How does this sound?

```diff
-#define B_RESTORE_HELD_BATTLE_ITEMS TRUE       // In Gen9, all non-berry items are restored after battle.
+#define B_RESTORE_HELD_BATTLE_ITEMS TRUE       // If TRUE, all non-berry items are restored after battle like in Gen9.
```

I think it's important to retain mention of the fact that this is a toggle for a feature that comes from the official games.

Reply:
In this case I would use gen_latest as we do usually with new gen mechanics.

Reply:
> In this case I would use gen_latest as we do usually with new gen mechanics.

But this is still a On/Off toggle in the codebase...

I don't know, doing something like:

```diff
-if (B_RESTORE_HELD_BATTLE_ITEMS == TRUE || gBattleStruct->itemLost[i].stolen)
+if (B_RESTORE_HELD_BATTLE_ITEMS == GEN_LATEST || gBattleStruct->itemLost[i].stolen)
```

doesn't really make much sense to me because the config truly decides whether a very specific feature is enabled or disabled.

Reply:
That's fair and I agree but this is the case for a lot of stuff where we used gen latest. Basically 50% of configs in this file are on/off switches. 

`B_RESTORE_HELD_BATTLE_ITEMS >= GEN_9`

Reply:
Shrug then. I guess I'll do that.